common/src/main/java/earth/terrarium/cadmus/common/network/packets/serverbound/RequestClaimSettingsPacket.java

```diff
@@ -0,0 +1,39 @@
+package earth.terrarium.cadmus.common.network.packets.serverbound;
+
+import com.teamresourceful.bytecodecs.base.ByteCodec;
+import com.teamresourceful.resourcefullib.common.network.Packet;
+import com.teamresourceful.resourcefullib.common.network.base.NetworkHandle;
+import com.teamresourceful.resourcefullib.common.network.base.PacketType;
+import com.teamresourceful.resourcefullib.common.network.base.ServerboundPacketType;
+import com.teamresourceful.resourcefullib.common.network.defaults.CodecPacketType;
+import com.teamresourceful.resourcefullib.common.utils.Scheduling;
+import com.teamresourceful.resourcefullib.common.utils.TriState;
+import earth.terrarium.cadmus.Cadmus;
+import earth.terrarium.cadmus.api.protections.ProtectionApi;
+import earth.terrarium.cadmus.api.teams.TeamApi;
+import earth.terrarium.cadmus.common.network.NetworkHandler;
+import earth.terrarium.cadmus.common.network.packets.clientbound.SyncClaimSettingsPacket;
+import earth.terrarium.cadmus.common.network.packets.clientbound.SyncClaimsPacket;
+import earth.terrarium.cadmus.common.utils.CadmusSaveData;
+import earth.terrarium.cadmus.common.utils.ModUtils;
+
+import java.util.HashMap;
+import java.util.concurrent.TimeUnit;
+
+public record RequestClaimSettingsPacket() implements Packet<RequestClaimSettingsPacket> {
+    public static final ServerboundPacketType<RequestClaimSettingsPacket> TYPE = CodecPacketType.Server.create(
+        Cadmus.id("request_claim_settings"),
+        ByteCodec.unit(RequestClaimSettingsPacket::new),
+        NetworkHandle.handle((packet, player) -> {
+            var settings = ProtectionApi.API.getSettings().stream().filter(
```

Comment:
can this be changed from functional to just a map that is foreached because this is very annoying to read